sdk/state/open.go

```diff
@@ -114,7 +125,10 @@ type OpenParams struct {
 	StartingSequence           int64
 }
 
-func (c *Channel) openTxs(d OpenAgreementDetails) (decl, close, formation *txnbuild.Transaction, err error) {
+func (c *Channel) openTxs(d OpenAgreementDetails) (txs OpenAgreementTransactions, err error) {
```

Comment:
💡 same thought here about adding a method comment similar to `closeTxs()`

Reply:
💯 130b932